changelogs/fragments/1057-module_zos_mvs_raw_errors_with_long_multi_line_quoted_string_in_content_field.yml

```diff
@@ -0,0 +1,6 @@
+bugfixes:
+  - zos_mvs_raw - module wasn't able to wrote properly a JCL with multi-line because truncated
+    the first one or two characters writing an syntactically incorrect JCL.
+    Change now validate the length of lines and allow multi-line declaration with | or |1-9 by added
+    two spaces to be syntactically corrects.
```

Comment:
suggestion:
  - zos_mvs_raw - module wasn't able to wrIte a multi-line JCL because it truncated
    the first one or two characters, resulting in syntactically incorrect JCL.
    Change now validates the length of lines and allow multi-line declaration with | or |1-9 by adding
    two spaces to be syntactically corrects.